### ISSUE TITLE
API-only tests

### ISSUE THREAD

**Issue by cavis** [PRX/meta.prx.org#65]
Move some of the current "wait for" tests in Publish to be API only.  For speeeeed.

Example: instead of the Publish tests publishing a story and waiting for it to show up in Feeder, just use the CMS api to directly create/publish that story-json, and watch Feeder.

**Comment by pkarman** [comment]
I appreciate the desire for speed. Does bypassing the UI negate the integration test aspect though?

**Comment by cavis** [comment]
Yeah, agreed.  I would probably vote for #64 to address speed before this one.  We may get to a point where API-only integration testing makes sense (like supporting 3rd party apps on our APIs)... but not there yet.
